test/e2e_node/runner/remote/run_remote.go

```diff
@@ -218,35 +216,36 @@ func main() {
 	gceImages := &internalImageConfig{
 		images: make(map[string]internalGCEImage),
 	}
+	// Parse images from given YAML file and convert them to internalGCEImage.
```

Comment:
doesn't have to be yaml, could be json.
"parse image from given config file and ...."